nixos/modules/services/hardware/udisks2.nix

```diff
@@ -28,6 +29,28 @@ in
         '';
       };
 
+      mountAtMedia = mkOption {
```

Comment:
I think we should call this something different... but let's see how the conversation about `/media` vs `/run/media` goes.

Reply:
Any suggestions?

Reply:
1. Why such an option to begin with? It became so popular?
2. Why just stop it here? Why not make it configurable? Why not allow me to mount it on `/media/pendrives/` or anything the user would like?

This way, a good name would be "udiskMountRoot" or "mountRoot"...

Reply:
It can't be done. `udisks2` only allows for `/media` or `/run/media/user`.